### PR TITLE
Enhanced the filter param of the table aws_ssm_inventory and fixed the error for the table aws_ssm_inventory_entry by type name

### PR DESCRIPTION
**Pull request by ParthaI**
# Integration test logs
<details>
  <summary>Logs</summary>

```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>

```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

**Review by atfurman — APPROVED** [review]
Functionally, this change does _exactly_ what we hoped it would. Have already developed and tested the desired reporting functionality using this feature branch, and now only await this feature in an official release to proceed with our feature release: https://github.com/stackArmor/threatalert-security-toolbox/issues/16

**Comment by ParthaI** [comment]
Thank you very much, @atfurman, for taking the time to test this pull request and for providing your valuable feedback. I will coordinate with the release team to ensure the integration of this feature. 👍

File: aws/table_aws_ssm_inventory.go [hunk 1]
```diff
@@ -44,6 +77,198 @@ func tableAwsSSMInventory(_ context.Context) *plugin.Table {
...
+			},
+			{
+				Name:        "component_attribute_value",
+				Description: "The value for the component attribute key.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(getFilterKeyWithOperator, "component_attribute_value"),
+			},
+			{
+				Name:        "application_attribute_key",
+				Description: "",
```
Comment (bigdatasourav) [inline aws/table_aws_ssm_inventory.go:106]: @ParthaI, Could you please add a description for the columns where it is missing?

File: aws/table_aws_ssm_inventory.go [hunk 2]
```diff
@@ -102,8 +327,17 @@ func listAwsSSMInventories(ctx context.Context, d *plugin.QueryData, _ *plugin.H
 	}
 
 	maxItems := int32(50)
-	input := buildSSMInventoryFilter(ctx, d.Quals)
-
+	input := buildSSMInventoryFilter(ctx, d)
+	// filter := types.InventoryFilter{
```
Comment (bigdatasourav) [inline aws/table_aws_ssm_inventory.go:331]: Please remove the commented code.

File: aws/table_aws_ssm_inventory.go (hunk 1 above)
Comment (ParthaI) [reply aws/table_aws_ssm_inventory.go]: Added the missing description for the columns.

File: aws/table_aws_ssm_inventory.go (hunk 2 above)
Comment (ParthaI) [reply aws/table_aws_ssm_inventory.go]: Cleaned-up the codes.

File: aws/table_aws_ssm_inventory.go [hunk 3]
```diff
@@ -44,6 +77,198 @@ func tableAwsSSMInventory(_ context.Context) *plugin.Table {
 				Description: "The type of inventory item returned by the request.",
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "filter_key",
+				Description: "The name of the filter key. Example: inventory filter key where managed node ID 'AWS:InstanceInformation.InstanceId'",
```
Comment (madhushreeray30) [inline aws/table_aws_ssm_inventory.go:82]:
```suggestion
				Description: "The name of the filter key. Example: inventory filter key where managed node ID 'AWS:InstanceInformation.InstanceId'.",
```

**Review by bigdatasourav — APPROVED** [review]

**Pull request merged**